src/cargo/core/features.rs

```diff
@@ -400,6 +400,36 @@ impl FromStr for FixEdition {
     }
 }
 
+/// The value for `-Zdetect-antivirus`.
+#[derive(Debug, Deserialize, Default, PartialEq, Eq, Hash, Copy, Clone)]
+#[serde(rename = "lowercase")]
```

Comment:
This isn't working correctly.

```suggestion
#[serde(rename_all = "lowercase")]
```